as/cNFT/assembly/royalty_payout.ts

```diff
@@ -4,7 +4,6 @@ import { persistent_tokens } from './models/persistent_tokens'
 import { persistent_tokens_royalty } from './models/persistent_tokens_royalty'
 import { Payout, TokenId } from './types'
 
```

Comment:
This will be exported, maybe we can merge for now until we sort out this `@nearBindgen` blocker?

Reply:
I can call `nft_payout` using cross contract calls if that's okay?

Reply:
No, it's fine. Just added a comment.